js/components/widgets/Link.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * @function Link
+ * @typedef {Object}
+ * @param {Object} data 
+ * @param {string} parrentClassName 
+ * @param {string} description 
+ * @param {string} url 
+ * @returns {string} HTML or empty 
+ */
+
+export const Link = (url, description, parrentClassName) => {
+  if(!Link) return '';
```

Comment:
What is this line for?

Reply:
> What is this line for?

line 12 says that if the function does not take a reference, then the function should not continue to perform its actions, but should return an empty string

Reply:
Link is your function name
You don't have the Link parameter!!!